Makefile

```diff
@@ -268,17 +268,17 @@ TARGET_MAX_CHAR_NUM := 15
 
 ## Show help
 help:
-	@echo ''
-	@echo 'Usage:'
-	@echo '  ${YELLOW}make${RESET} ${GREEN}<target>${RESET}'
-	@echo ''
-	@echo 'Targets:'
-	@awk '/^[a-zA-Z\-\_0-9]+:/ { \
-	  helpMessage = match(lastLine, /^## (.*)/); \
-	  if (helpMessage) { \
-	    helpCommand = substr($$1, 0, index($$1, ":")-1); \
-	    helpMessage = substr(lastLine, RSTART + 3, RLENGTH); \
-	    printf "  ${YELLOW}%-$(TARGET_MAX_CHAR_NUM)s${RESET} ${GREEN}%s${RESET}\n", helpCommand, helpMessage; \
-	  } \
-	} \
-	{ lastLine = $$0 }' $(MAKEFILE_LIST)
+  @echo ''
+  @echo 'Usage:'
+  @echo '  ${YELLOW}make${RESET} ${GREEN}<target>${RESET}'
+  @echo ''
+  @echo 'Targets:'
+  @awk '/^[a-zA-Z\-\_0-9]+:/ { \
+    helpMessage = match(lastLine, /^## (.*)/); \
+    if (helpMessage) { \
+      helpCommand = substr($$1, 0, index($$1, ":")-1); \
+      helpMessage = substr(lastLine, RSTART + 3, RLENGTH); \
+      printf "  ${YELLOW}%-$(TARGET_MAX_CHAR_NUM)s${RESET} ${GREEN}%s${RESET}\n", helpCommand, helpMessage; \
+    } \
+  } \
+  { lastLine = $$0 }' $(MAKEFILE_LIST)
```

Comment:
Are you sure we want to change the whitespaces across this whole big makefile? It's going to be a pain to maintain if we do any comparisons with original forked repo?

Reply:
Thanks for pointing this out, I've switched GH Actions for this repo to our generic workflow that doesn't use Makefile. I've reverted my changes to Makefile